src/kademlia/dht_storage.cpp

```diff
@@ -346,6 +345,7 @@ namespace
 		}
 
 		void put_immutable_item(sha1_hash const& target
+			, sha1_hash const& our_id
```

Comment:
won't this cause odd prioritizations between the two nodes? since "out ID" will be different depending on who's putting. presumably the ideal would be to prioritize based on the distance to _any_ of our node IDs. Not sure what the best way to do that is though..